### PR TITLE
Fix configuration issues for branch protection

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml [hunk 1]
```diff
@@ -65,6 +70,18 @@ presubmits:
...
     run_if_changed: ^roles/openshift_logging.*|playbooks/.+/openshift-cluster/openshift.logging.*
     trigger: ((?m)^/test( all| logging),?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - release-3.7
+    context: ci/openshift-jenkins/tox
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_openshift_ansible_tox_310
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml]: nitpick: name update?

Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml]: that's what the jenkins job is named and I'm not changing that

Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml]: it just needs to be optional on the branches where protection is off

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.9-presubmits.yaml [hunk 2]
```diff
@@ -39,6 +42,18 @@ presubmits:
...
     run_if_changed: ^roles/openshift_logging.*|playbooks/openshift-logging/.*
     trigger: ((?m)^/test( all| logging),?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - release-3.9
+    context: ci/openshift-jenkins/tox
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_openshift_ansible_tox_310
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.9-presubmits.yaml]: nitpick: name update?

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.7-presubmits.yaml]: Right, makes sense

**Pull request merged**